client/my-sites/plans-v2/utils.ts

```diff
@@ -119,6 +119,12 @@ export function durationToString( duration: Duration ): DurationString {
 }
 
 export function durationToText( duration: Duration ): TranslateResult {
+	if ( 'i5' === getJetpackCROActiveVersion() ) {
+		return duration === TERM_MONTHLY
+			? translate( 'per month{{br/}}billed monthly', { components: { br: createElement( 'br' ) } } )
+			: translate( 'per month{{br/}}billed yearly', { components: { br: createElement( 'br' ) } } );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( '%1$s per month, billed yearly', { context: 'Describes the amount of money per month, e.g. "$10 per month, billed yearly"'} )` *ES Score: 9*